### DISCUSSION TITLE
License Agreement

### DISCUSSION THREAD

**Discussion by Lednerb** [wdelfuego/nova-calendar#22] [Ideas & Feature Requests]
@wdelfuego 
Just as a quick link to a piece of software that I've saw in the past and maybe helps with the License Agreement:

https://github.com/cla-assistant

https://cla-assistant.io/

This should integrate nicely with GitHub and whenever someone opens a PR there is a check and the contributor can accept / sign the repo's license.

On the second link you'll find a preview demo of the process.

Maybe it's worth integrating it.

**Comment by wdelfuego** [comment]
Looks good! 👍

**Comment by wdelfuego** [comment]
Closing this for now; I've since learned that GitHub users implicitly agree to contributor licenses by creating pull requests and this package has so few contributors that I don't want to invest time in making this process any more complex than it needs to be.

Thanks for the input, @Lednerb, hope you're still a happy Nova Calendar user :).
